backend/src/test/resources/queries/add-multiplex-result-mutation

```diff
@@ -1,17 +0,0 @@
-mutation addMultiplexResult($deviceId: String!, $results: [MultiplexResultInput]!, $patientId: ID!, $dateTested: DateTime) {
```

Comment:
this was moved, not deleted, [see this comment](https://github.com/CDCgov/prime-simplereport/pull/4243/files#r969948477)